lte/gateway/configs/templates/mme.conf.template

```diff
@@ -75,19 +79,19 @@ MME :
     # maximum of 16 TAIs, comma separated
     # !!! Actually use only one PLMN
     TAI_LIST = (
-    {% if not attachedEnodebTacs|length %}
-         { MCC="{{ mcc }}" ; MNC="{{ mnc }}" ; TAC="{{ tac }}"; }
-    {%- else %}
-         {% for enbTac in attachedEnodebTacs -%}
-         { MCC="{{ mcc }}" ; MNC="{{ mnc }}" ; TAC="{{ enbTac }}"; }{% if not loop.last %},{% endif %}
-         {% endfor -%}
-    {%- endif %}
+        { MCC: "001"; MNC: "01"; TAC: "1" },
+        { MCC: "001"; MNC: "02"; TAC: "2" },
+        { MCC: "001"; MNC: "03"; TAC: "3" },
+        { MCC: "001"; MNC: "04"; TAC: "4" },
+        { MCC: "001"; MNC: "05"; TAC: "5" }
     );
 
     TAC_LIST = (
-         {% for enbTac in attached_enodeb_tacs -%}
-         { MCC="{{ mcc }}" ; MNC="{{ mnc }}" ; TAC="{{ enbTac }}"; }{% if not loop.last %},{% endif %}
-         {% endfor %}
+        { MCC: "001"; MNC: "01"; TAC: "1" },
+        { MCC: "001"; MNC: "02"; TAC: "2" },
+        { MCC: "001"; MNC: "03"; TAC: "3" },
+        { MCC: "001"; MNC: "04"; TAC: "4" },
+        { MCC: "001"; MNC: "05"; TAC: "5" }
     );
```

Comment:
This file looks like added by accident here.

Reply:
reverted back